jans-auth-server/client/src/main/java/io/jans/as/client/RegisterRequest.java

```diff
@@ -1790,4 +1938,28 @@ public String getRedirectUrisRegex() {
     public void setRedirectUrisRegex(String redirectUrisRegex) {
         this.redirectUrisRegex = redirectUrisRegex;
     }
+
+    private void addClientMetadataParam(Map<String, String> parameters, RegisterRequestParam registerRequestParam, ClientMetadataValue clientMetadataValue) {
+        if (clientMetadataValue != null && clientMetadataValue.size() > 0) {
+            for (String languageTag : clientMetadataValue.getLanguageTags()) {
+                if (StringUtils.isBlank(languageTag)) {
+                    parameters.put(registerRequestParam.toString(), clientMetadataValue.getValue());
+                } else {
+                    parameters.put(registerRequestParam.toString() + "#" + languageTag, clientMetadataValue.getValue(languageTag));
+                }
+            }
+        }
+    }
+
+    private void addClientMetadataParam(JSONObject parameters, RegisterRequestParam registerRequestParam, ClientMetadataValue clientMetadataValue) {
```

Comment:
If add `addToMap()` method then this I guess can be tricked as (pseudo code that has to be checked.): 

`new JSONObject(localicalizedString.addToMap(jsonObject.ToMap(), registerRequestParam.toString()));`

Advantage is that we can re-use `addToMap` in other places.